### DISCUSSION TITLE
Messages public property conflict

### DISCUSSION THREAD

**Discussion by JohnnyEvo** [livewire/livewire#7064] [7. Bugs (archived)]
### Livewire version

v3.0.8

### Which PHP version are you using?

PHP 8.2

### Repository URL

https://github.com/JohnnyEvo/livewire/commits/bug/MessagesProperty

### Steps To Reproduce

1. go to /form page
2. submit empty form
3. error displayed

### Additional information

I don't know if bug known or missing documentation, but we can't used `messages` public property in components because this name property is in conflicts with validation form.

```php
<?php

namespace App\Livewire;

use Livewire\Attributes\Rule;
use Livewire\Component;

class Form extends Component
{
    #[Rule('required')]
    public $messages = '';
    public $status = 'Not Saved.';

    public function save()
    {
        $this->validate();
        $this->status = 'Saved.';
    }

    public function render()
    {
        return view('livewire.form');
    }
}
```

```html
<div>
    <h1>{{ $this->status }}</h1>
    <form wire:submit="save">
        <input type="text" wire:model="messages">

        <button type="submit">Save</button>
    </form>
</div>
```

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by lrljoe** [comment]
Please don't link to repos for people to look at a bug, I suspect that this is a duplicate of the other form attributes discussions?
